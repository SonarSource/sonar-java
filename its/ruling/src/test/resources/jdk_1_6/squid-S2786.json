{
'jdk_1_6:java/awt/Desktop.java':[
72,
],
'jdk_1_6:java/awt/Dialog.java':[
130,
226,
],
'jdk_1_6:java/awt/EventFilter.java':[
15,
],
'jdk_1_6:java/awt/MultipleGradientPaint.java':[
29,
51,
],
'jdk_1_6:java/security/KeyRep.java':[
49,
],
'jdk_1_6:java/text/Normalizer.java':[
108,
],
}
