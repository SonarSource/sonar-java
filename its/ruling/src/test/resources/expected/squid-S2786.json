{
'project:com/sun/java/swing/plaf/gtk/GTKEngine.java':[
65,
94,
],
'project:com/sun/java/swing/plaf/windows/TMSchema.java':[
64,
87,
224,
472,
528,
],
'project:java/awt/Desktop.java':[
90,
],
'project:java/awt/Dialog.java':[
144,
240,
],
'project:java/awt/EventFilter.java':[
33,
],
'project:java/awt/GraphicsDevice.java':[
122,
],
'project:java/awt/MultipleGradientPaint.java':[
47,
69,
],
'project:java/awt/Window.java':[
165,
],
'project:java/awt/font/NumericShaper.java':[
162,
],
'project:java/lang/Character.java':[
3076,
],
'project:java/lang/invoke/FilterGeneric.java':[
283,
],
'project:java/security/KeyRep.java':[
66,
],
'project:java/security/cert/CertPathValidatorException.java':[
260,
],
'project:java/text/Normalizer.java':[
126,
],
'project:javax/imageio/ImageIO.java':[
431,
],
'project:javax/swing/MultiUIDefaults.java':[
138,
],
'project:javax/swing/plaf/nimbus/AbstractRegionPainter.java':[
490,
],
'project:javax/swing/text/html/StyleSheet.java':[
1953,
],
}
